stdlib/LinearAlgebra/src/triangular.jl

```diff
@@ -157,6 +159,9 @@ imag(A::UnitUpperTriangular) = UpperTriangular(triu!(imag(A.data),1))
 Array(A::AbstractTriangular) = Matrix(A)
 parent(A::AbstractTriangular) = A.data
 
+copyto!(dest::SymTridiagonal, src::SymTridiagonal) =
+    (copyto!(dest.dv, src.dv); copyto!(dest.ev, src.ev); dest)
+
```

Comment:
Should this better be moved to `tridiag.jl`?

Reply:
Yes :-)
I moved it